llap-common/src/test/org/apache/hadoop/hive/llap/AsyncResponseHandlerTest.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.llap;
+
+import com.google.protobuf.Message;
+import org.apache.hadoop.hive.llap.AsyncPbRpcProxy.ExecuteRequestCallback;
+import org.apache.hadoop.util.concurrent.AsyncGet;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Thread.sleep;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class AsyncResponseHandlerTest {
```

Comment:
Class name should start with Test -> TestAsyncResponseHandler

Reply:
I've seen both styles used throughout the codebase. I checked the jenkins run and saw the test ran fine: http://ci.hive.apache.org/job/hive-precommit/view/change-requests/job/PR-1291/7/testReport/org.apache.hadoop.hive.llap/AsyncResponseHandlerTest/
I see both styles used in the codebase.

Surefire plugin doc says both styles are ok: https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html

Reply:
Hey @mustafaiman  -- I remember spending time in the past for a test that did not trigger because of naming and wanted to make sure we dont hit that issue.

You are right though, both Test* and *Test should be fine and since it run fine we are good :)